peercrawler_html_server.py

```diff
@@ -110,10 +111,12 @@ def logs():
 
 
 def main():
-    setup_logger(logger)
+    args = peercrawler.parse_args()
```

Comment:
You cannot just use the peercrawler command line arguments exactly.
They are similar but not the same.
For example, -c, -f, -s do not apply and -p would be amiguous since there are 2 different ports that the html peercrawler listen to.

Reply:
Well, the not applicable ones wouldn't be a problem, as they would be simply ignored, as for the port parameter I agree that it would be ambiguous. I'll change it.

Reply:
They are a problem when you use -h to display the options accepted.